stack.go

```diff
@@ -83,6 +84,18 @@ func (f Frame) Format(s fmt.State, verb rune) {
 	}
 }
 
+// MarshalJSON formats a stacktrace Frame for easy inclusion in JSON stack
+// traces. The format is the same as that of fmt.Format("%+v"), but without
+// newlines or tabs.
+func (f Frame) MarshalJSON() ([]byte, error) {
```

Comment:
Rather than just marshaling this to a string, then `json.Marshal`ing that, I think we would prefer `MarshalText` here.

From [the encoding/json references](https://golang.org/pkg/encoding/json/#Marshal) `If no MarshalJSON method is present but the value implements encoding.TextMarshaler instead, Marshal calls its MarshalText method and encodes the result as a JSON string.`

Reply:
Good suggestion.